kolibri/core/assets/src/core-actions.js

```diff
@@ -349,6 +443,85 @@ function stopTrackingProgress(store, Kolibri) {
   updateTimeSpent(store, Kolibri, true);
 }
 
+function saveMasteryLog(store, Kolibri) {
```

Comment:
do we expect this function (and `saveAttemptLog`) to be re-used? If not, might improve readability to inline them into `saveAssessmentLogs`.